### PR TITLE
[ISC] Document dhcpd/dhcpcd CVE-2021-25217

### PR DESCRIPTION
**Pull request by isc-security-officer**

### CODE REVIEW

File: 2021/25xxx/CVE-2021-25217.json [hunk 1]
```diff
@@ -1,18 +1,124 @@
 {
```
Comment (cve-team) [inline 2021/25xxx/CVE-2021-25217.json:1]: 'value' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/25xxx/CVE-2021-25217.json (hunk 1 above)
Comment (isc-security-officer) [reply 2021/25xxx/CVE-2021-25217.json:1]: Oops.  I'll rework and submit a new MR.

Comment (isc-security-officer) [reply 2021/25xxx/CVE-2021-25217.json:1]: I've removed the extraneous empty problemtype_data/description

**Pull request merged**